app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, "Help") %>
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
+  <a href="http://www.railstutorial.org/help">Rails Tutorial help page</a>.
+  To get help on this sample app, see the
+  <a href="http://www.railstutorial.org/book"><em>Ruby on Rails Tutorial</em>
+  book</a>.
+</p>
```

Comment:
em thấy thường sẽ có 1 line ở cuối mỗi file thì sẽ không báo end-of-file, em có thêm vào 2 file help.html.erb và contact.html.erb nhưng comment vẫn còn đây là sao chị nhỉ